class ValidationError(Exception):
    pass

class DataAccessError(Exception):
    pass